### PR TITLE
chore: removed RSA1_5 as default enc alg

### PR DESCRIPTION
**Pull request by peppelinux**

### CODE REVIEW

**Review by jschlyter — DISMISSED** [review]
Perhaps the commit should say what we replace it with?
Would it make sense to also move from CBC to GCM?

File: src/cryptojwt/jwt.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def __init__(
         sign_alg="RS256",
         encrypt=False,
         enc_enc="A128CBC-HS256",
```
Comment (peppelinux) [inline src/cryptojwt/jwt.py]: @jschlyter can we put `A128GCM` instead of `A128CBC-HS256`

did you mean this?

Comment (jschlyter) [reply src/cryptojwt/jwt.py]: Yes

Comment (peppelinux) [reply src/cryptojwt/jwt.py]: Done here
https://github.com/IdentityPython/JWTConnect-Python-CryptoJWT/pull/125/commits/619f9ef2ed60b6e95af02c9269a98e09922ed834

**Review by jschlyter — APPROVED** [review]
Yes, much better defaults. Are there any interoperability issues that we are aware of @rohe ?

**Comment by rohe** [comment]
Not that I know of.

**Review by rohe — APPROVED** [review]

**Pull request merged**